main/Game_State.py

```diff
@@ -0,0 +1,136 @@
+import pygame
+import os
+import configparser
+from debug_log import Debug
+from tools import debug_log
+from gui.button import Button
+from character import Character
+
+
+class GameState:
+
+    def main_menu(self):
+        config = configparser.ConfigParser()
+        config.read("config.ini")
+
+        screen_width = config.getint("General", "screen_width")
+        screen_height = config.getint("General", "screen_height")
+        fps = config.getint("General", "FPS")
+        title = config.get("General", "title")
+
+        screen_size = (screen_width, screen_height)
+        win = pygame.display.set_mode(screen_size)
+        background = pygame.transform.scale(pygame.image.load(os.path.join('..', 'assets', 'background.jpg')),
+                                            screen_size)
+        pygame.display.set_caption(title)
```

Comment:
Elements that will not change during runtime, as for sure `config.ini` or maybe even screen dimensions could be placed in `__init__` instead of placing it in each method in this class (because it seems like each will use them)

Another solution might be set configparser outside the class and then use it inside, thats just suggestions to avoid repetitions.

```suggestion
class GameState:
    def __init__(self):
        self.config = configparser.ConfigParser()
        self.config.read("config.ini")

        self.screen_width = self.config.getint("General", "screen_width")
        self.screen_height = self.config.getint("General", "screen_height")
        self.fps = self.config.getint("General", "FPS")
        self.title = self.config.get("General", "title")

        self.screen_size = (self.screen_width, self.screen_height)
        
    def main_menu(self):
        win = pygame.display.set_mode(screen_size)
        background = pygame.transform.scale(pygame.image.load(os.path.join('..', 'assets', 'background.jpg')),
                                            screen_size)
        pygame.display.set_caption(title)
```